modules/avm/variables.tf

```diff
@@ -65,6 +65,12 @@ variable "region" {
   description = "The default region of the account"
 }
 
+variable "ssh_key_id" {
+  type        = string
+  default     = null
+  description = "The SSH key ID to assign to the workspace"
```

Comment:
I never really liked this description, I think we should make it more clear: "The SSH key ID to assign to the Terraform Cloud workspace"

Reply:
Updated :)